src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManagerRecoverabilityPolicy.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NServiceBus
+{
+    using Transport;
+
+    class TimeoutManagerRecoverabilityPolicy : IRecoverabilityPolicy
+    {
+        public TimeoutManagerRecoverabilityPolicy()
+        {
+            RaiseNotifications = false;
+        }
+
+        public RecoverabilityAction Invoke(ErrorContext errorContext)
+        {
+            if (errorContext.NumberOfDeliveryAttempts <= MaxNumberOfFailedRetries)
+            {
+                return RecoverabilityAction.ImmediateRetry();
+            }
+
+            return RecoverabilityAction.MoveToError();
+        }
+
+        public bool RaiseNotifications { get; }
```

Comment:
I'm not to enthusiastic about that approach with the RaiseNotificiations property. Should we at least rename it to `RaiseRecoverabilityNotifications` to make it a bit more clearer that this is mainly about the flr/slr/errorq event and not potential internal or other events?